stable/filebeat/templates/daemonset.yaml

```diff
@@ -117,12 +121,25 @@ spec:
         hostPath:
           path: /var/lib/docker/containers
       - name: filebeat-config
-        secret:
```

Comment:
Why would we make this change?  Passwords can be in the filebeat configuration?

Reply:
Wrapping everything into a secret is not the best thing to do, imo... There's https://www.elastic.co/guide/en/beats/filebeat/6.5/keystore.html which should be used for secrets. Though I haven't given it a shot this time.

Reply:
Until something like that is implemented in this chart we can't merge your changes as it would force everyone to have passwords in the config map.

Reply:
I agree. Config files with passwords should be in a secret.
Is there any other benefit of using a ConfigMap for this?